src-ui/src/manifest.webmanifest

```diff
@@ -5,10 +5,14 @@
   "icons": [
     {
       "src": "favicon.ico",
-      "sizes": "128x128"
+      "sizes": "256x256"
+    },
+    {
+      "src": "assets/logo-notext.svg",
+      "sizes": "any"
     }
   ],
-  "name": "Paperless NG",
-  "short_name": "Paperless NG",
+  "name": "Paperless NGX",
+  "short_name": "Paperless NGX",
```

Comment:
In most parts the branding is "Paperless-ngx" as far as I'm aware.

Is there an official variant?

Reply:
In the title it is "Paperless-ngx"
https://github.com/paperless-ngx/paperless-ngx/blob/666b938550963d136a4f2274cafc0d8d14993761/src-ui/src/index.html#L5

Reply:
I changed the title accordingly. Thank you for the suggestion!